app/assets/javascripts/govuk.js

```diff
@@ -24,27 +24,10 @@ $(document).ready(function () {
   var showHide = new ShowHide()
   showHide.init()
 
-  var locationInput = document.getElementById('location')
-  if (locationInput) {
-    accessibleAutocomplete({
-      element: locationInput,
-      id: 'location',
-      name: locationInput.getAttribute('name'),
-      source: suggest_locations,
-      showAllValues: true, dropdownArrow: () => '',
-    })
-  }
 
+  addAutocomplete('#publisher', 'publisher', publisherTitles)
```

Comment:
Do you think we should split this out as a module and put it in its own file in `app/assets/javascripts/`